content/en/docs/azure/authentication-oidc.md

```diff
@@ -148,12 +148,24 @@ This section shows the how to set up Kubeflow with authentication and authorizat
     kubectl patch service -n istio-system istio-ingressgateway -p '{"spec": {"type": "LoadBalancer"}}'
     ```
 
-   After that, obtain the `LoadBalancer` IP or Hostname from its status and create the necessary certificate.
+    After that, obtain the `LoadBalancer` IP or Hostname from its status and create the necessary certificate.
 
     ```
     kubectl get svc -n istio-system istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0]}'
     ```
 
+    {{% alert title="Warning" color="warning" %}}
+    If you are exposing Ingress gateway through public IP please ensure its matches IP address of OIDC ```REDIRECT_URL```, by running:
+    ```
+    kubectl get statefulset authservice -n istio-system -o yaml
+    ```
+    If it doesn't match, update ```REDIRECT_URL``` in the statefulset to be the public IP address you got in the last step, by running:
+        ```
+    kubectl edit statefulset authservice -n istio-system
+    ```
```

Comment:
Suggestions:

- "make sure it matches" <- "please ensure its matches"
- "the" + "IP address" (definite article)
- To help new readers: 
  - Add a link to **Ingress** from the k8s site, just in case (gateway ingress is talked about on Microsoft's site but Ingress comes from k8s, as far as I understand
  - **StatefulSet** should be spelled out with capital "S"s (https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/) - add a link just case readers aren't aware
- Use single backticks for in-line "code font" (e.g. `REDIRECT_URL`) instead of three backticks
  - This could be useful: https://kubernetes.io/docs/contribute/style/style-guide/#use-code-style-for-filenames-directories-and-paths
- Separate paragraphs and code blocks with blank lines
- "from the last step" <-"you got in the last step" (too informar)

```suggestion
    If you are exposing [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) gateway through the public IP, make sure it matches the IP address of OIDC `REDIRECT_URL` by running:
    
    ```shell
    kubectl get statefulset authservice -n istio-system -o yaml
    ```

    If it doesn't match, then update the `REDIRECT_URL` in the [StatefulSet](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/) to be the public IP address from the last step, by running:
    
    ```
    kubectl edit statefulset authservice -n istio-system
    ```

```